Toolset/libraries/revidelibrary.8.livecodescript

```diff
@@ -9585,36 +9589,36 @@ command revIDESuspendDevelopmentTools
    -- Remove front and back scripts that pertain to the IDE
    -- If the shift-key is down, we remove *all* 'rev' front and back scripts, including
    -- the library scripts.
-   local tScriptsList, tRemoveFrontList, tRemoveBackList
+   local tScriptsList, tRemoveFrontList, tRemoveBackList, tLine
    select empty
    if the shiftKey is down then
       put the frontScripts into tScriptsList
-      repeat for each line l in tScriptsList
-         if char 1 to 3 of the short name of l is "rev" then
-            put l & cr after tRemoveFrontList
-            remove script of l from front
+      repeat for each line tLine in tScriptsList
+         if revIDEObjectIsOnIDEStack(tLine) then
+            put tLine & cr after tRemoveFrontList
+            remove script of tLine from front
          end if
       end repeat
       put the backScripts into tScriptsList
-      repeat for each line l in tScriptsList
-         if char 1 to 3 of the short name of l is "rev" then
-            put l & cr after tRemoveBackList
-            remove script of l from back
+      repeat for each line tLine in tScriptsList
+         if revIDEObjectIsOnIDEStack(tLine) then
+            put tLine & cr after tRemoveBackList
+            remove script of tLine from back
          end if
       end repeat
    else
       put the frontScripts into tScriptsList
-      repeat for each line l in tScriptsList
-         if the short name of l is among the items of "revFrontScriptLibrary,revDebugger,revShortCutsLibrary" then
-            put l & cr after tRemoveFrontList
-            remove script of l from front
+      repeat for each line tLine in tScriptsList
+         if the short name of tLine is among the items of "revFrontScriptLibrary,revDebugger,revShortCutsLibrary" then
```

Comment:
I don't think there's a stack called `revDebugger` anymore, so that can probably be removed!

Reply:
The frontscript is called revdebuggerlibrary now, so if the intent is to remove it from the environment as well the name should be changed here. Also, for consistency in case it matters on some platform, the stack names should be all lowercase.